src/main/java/com/excella/foodinc/controllers/FoodController.java

```diff
@@ -0,0 +1,38 @@
+package com.excella.foodinc;
+
+import java.util.HashMap;
+import java.util.Map;
+import reactor.core.publisher.Mono;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class FoodController {
+
+    private static Map<String, Food> foodRepo = new HashMap<>();

```

Comment:
I would move all references to food (either real data or fake data) out of the Resource layer. The controller should not be concerned with the underlying implementation. Remember R-S-D.